libraries/radspreadprocessing/features/hyperlink.md

```diff
@@ -72,13 +72,23 @@ __Example 2__ creates a hyperlink to a cell range somewhere in the document.
 {{endregion}}
 
 
+__Example 3__ creates a hyperlink to a cell in a another worksheet of the document.
```

Comment:
__Example 3__ creates a hyperlink to a cell in another worksheet of the document.